src/bpf/profiler.h

```diff
@@ -172,20 +166,19 @@ typedef struct {
 typedef struct {
   int task_id;
   int pid;
-  unsigned long long user_stack_id;
-  unsigned long long kernel_stack_id;
-} stack_count_key_t;
+} stack_key_t;
 
 typedef struct {
   native_stack_t stack;
+  native_stack_t kernel_stack;
```

Comment:
Probably no need to add this here if we use a diff struct to send data to uspace.